core/src/main/scala/org/apache/spark/status/api/v1/StagesResource.scala

```diff
@@ -154,6 +154,29 @@ private[v1] class StagesResource extends BaseAppResource {
     }
   }
 
+  @GET
+  @Path("{stageId: \\d+}/{stageAttemptId: \\d+}/executorMetricsDistribution")
+  def executorSummary(
+      @PathParam("stageId") stageId: Int,
+      @PathParam("stageAttemptId") stageAttemptId: Int,
+      @DefaultValue("0.05,0.25,0.5,0.75,0.95")
```

Comment:
change the default value to 0.0,0.25,0.5,0.75,1.0.  In a parallel system, the duration of a stage is often determined by the slowest task/executor.  To monitor/debug a skew issue, the maximal value (or 100% percentile value) is more useful than the 95% percentile value.

Reply:
> change the default value to 0.0,0.25,0.5,0.75,1.0. In a parallel system, the duration of a stage is often determined by the slowest task/executor. To monitor/debug a skew issue, the maximal value (or 100% percentile value) is more useful than the 95% percentile value.

Add this value since 
https://github.com/apache/spark/blob/f54d430c536bfb4731401868960d6e35ddd089ea/core/src/main/scala/org/apache/spark/status/api/v1/StagesResource.scala#L72-L78

Similar API need keep consistent too....

Reply:
I see.  You want to keep it consistent.

Reply:
> I see. You want to keep it consistent.

Emmmm, I found that in ui and restful API the quantiles is not same....